config/webpack.config.dev.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+const webpack = require('webpack')
+
+module.exports = {
+  devtool: '#source-map',
+  externals: ['cozy'],
+  plugins: [
+    new webpack.DefinePlugin({
+      __SERVER__: JSON.stringify('http://app.cozy.local'),
```

Comment:
Is it wanted to use hardcoded server URL like here? (now we can use app.cozy.tools by the way)